src/main/scala/com/twitter/finagle/postgres/values/ValueEncoder.scala

```diff
@@ -189,6 +189,17 @@ object ValueEncoder extends LowPriorityEncoder {
     m: Map[String, String] => m.mapValues(Option(_))
   }
 
+  implicit val jsonb: ValueEncoder[Array[Byte]] = instance[Array[Byte]](
```

Comment:
This creates a second `ValueEncoder[Array[Byte]]` the other one being `bytea`. The fact that there are two with the same signature causes a compiler fail.